includes/data-stores/class-wc-admin-reports-stock-stats-data-store.php

```diff
@@ -0,0 +1,137 @@
+<?php
+/**
+ * WC_Admin_Reports_Stock_Stats_Data_Store class file.
+ *
+ * @package WooCommerce Admin/Classes
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * WC_Reports_Stock_Stats_Data_Store.
+ */
+class WC_Admin_Reports_Stock_Stats_Data_Store extends WC_Admin_Reports_Data_Store implements WC_Admin_Reports_Data_Store_Interface {
+
+	/**
+	 * Get stock counts for the whole store.
+	 *
+	 * @param array $query Not used for the stock stats data store, but needed for the interface.
+	 * @return array Array of counts.
+	 */
+	public function get_data( $query ) {
+		$report_data = array();
+		$low_stock_transient_name = 'wc_admin_stock_count_lowstock';
+		$low_stock_count          = get_transient( $low_stock_transient_name );
+		if ( false === $low_stock_count ) {
+			$low_stock_count = $this->get_low_stock_count();
```

Comment:
I think you'll need to check the low inventory thresholds set on individual items as well